### PR TITLE
Expose exemptfee to sdk-flutter

### PR DESCRIPTION
**Pull request by ubbabeck**
Update flutter sdk.
adding required exemptfee param  https://github.com/breez/breez-sdk/pull/440

### CODE REVIEW

File: libs/sdk-flutter/lib/breez_bridge.dart [hunk 1]
```diff
@@ -454,6 +454,7 @@ extension SDKConfig on Config {
     String? apiKey,
     double? maxfeePercent,
     NodeConfig? nodeConfig,
+    int? exemptfeeMsat,
```
Comment (ok300) [inline libs/sdk-flutter/lib/breez_bridge.dart:457]: The new arg comes before `nodeConfig`

File: libs/sdk-flutter/lib/breez_bridge.dart [hunk 2]
```diff
@@ -465,6 +466,7 @@ extension SDKConfig on Config {
       apiKey: apiKey ?? this.apiKey,
       maxfeePercent: maxfeePercent ?? this.maxfeePercent,
       nodeConfig: nodeConfig ?? this.nodeConfig,
+      exemptfeeMsat: exemptfeeMsat ?? this.exemptfeeMsat,
```
Comment (ok300) [inline libs/sdk-flutter/lib/breez_bridge.dart:469]: Same here

**Review by ok300 — CHANGES_REQUESTED** [review]
Overall looks good, just a few NITs.

File: libs/sdk-flutter/lib/breez_bridge.dart (hunk 1 above)
Comment (ubbabeck) [reply libs/sdk-flutter/lib/breez_bridge.dart:456]: updated. thanks :D

File: libs/sdk-flutter/lib/breez_bridge.dart (hunk 2 above)
Comment (ubbabeck) [reply libs/sdk-flutter/lib/breez_bridge.dart:468]: updated. thanks :D

**Review by ok300 — APPROVED** [review]

**Review by roeierez — APPROVED** [review]

**Pull request merged**